src/theme/Alert.ts

```diff
@@ -0,0 +1,85 @@
+import { AlertProps } from "@chakra-ui/react"
+
+const defaultProps = {
+  status: "info",
+  variant: "subtle",
+}
+
+const statusWarning = {
+  container: {
+    bg: "hsla(26, 96%, 65%, 10%)",
+    color: "hsla(0, 0%, 100%, 80%)",
+  },
+  icon: {
+    color: "#FB9950",
+  },
+}
+
+const statusInfo = {
+  container: {
+    color: "white",
+    bg: "#1E1E1E",
+  },
+  icon: {
+    color: "currentColor",
```

Comment:
I'm not sure what the "currentColor" syntax does tbh 🤔 I mean, I can assume based on it's name, but is it actually necessary to specify the color in such case?